src/ert/storage/migration/to9.py

```diff
@@ -0,0 +1,71 @@
+import json
+import os
+from pathlib import Path
+
+import polars
+
+info = "Migrate finalized response keys into configs"
+
+
+def _migrate_response_configs_wrt_finalized_keys(path: Path) -> None:
+    for experiment in path.glob("experiments/*"):
+        ensembles = path.glob("ensembles/*")
+
+        experiment_id = None
+        with open(experiment / "index.json", encoding="utf-8") as f:
+            exp_index = json.load(f)
+            experiment_id = exp_index["id"]
+
+        responses_config = None
+        with open(experiment / "responses.json", mode="r", encoding="utf-8") as f:
+            responses_config = json.load(f)
+            for response_type, config in responses_config.items():
+                if not config.get("has_finalized_keys"):
```

Comment:
```suggestion
                if "has_finalized_keys not in config:
```
just a small suggestion I at least think reads better